script/validators/source_validator.py

```diff
@@ -219,8 +223,31 @@ def check_namespaces(file_path):
     return status
 
 
+def check_includes(file_path):
+    """Checks whether local includes are done via #include<...>"""
+    with open(file_path, "r") as f:
+        path_pattern = re.compile(r'^#include <(.*?)>')
+        includes = os.listdir('src/include')
+        linenum = 0
+        status = True
+        for line in f:
+            linenum += 1
+            res = path_pattern.match(line)
+            if res:
+                path = res.groups()[0].lstrip("include/")
+                if path in PATHS:
+                    if status:
+                        LOG.info("Invalid include in %s" % file_path)
+                    status = False
+                    LOG.info("Line %s: %s" % (linenum, line))
```

Comment:
Please strip the line before printing it to avoid additional new lines. (cmp. https://github.com/schedutron/peloton/blob/84b8947b388b7686a3b4e19aef2be1c5fc03bba9/script/validators/source_validator.py#L152)